modules/tinymce/src/plugins/table/test/ts/browser/ClipboardTest.ts

```diff
@@ -732,4 +733,37 @@ describe('browser.tinymce.plugins.table.ClipboardTest', () => {
       '</tbody></table>'
     );
   });
+
+  it('TINY-6939: Pasting into a table should fire TableModified event', () => {
+    const editor = hook.editor();
+    let events: Array<TableEventData> = [];
+
+    const callback = (e: TableEventData) => {
+      events.push({
+        structure: e.structure,
+        style: e.style,
+      });
+    };
+
+    const clearEvents = () => events = [];
+    clearEvents();
```

Comment:
```suggestion
```
This is unnecessary here as we've only just initialised it so it's definitely going to be empty.

Reply:
Fixed.